compiler/astmsgs.nim

```diff
@@ -1,6 +1,6 @@
 # this module avoids ast depending on msgs or vice versa
 import std/strutils
-import options, ast, msgs
+import options, ast, msgs, types
```

Comment:
Instead of the recursive module dependency, move `addTypeDeclVerboseMaybe` to types.nim

Reply:
done